packages/next/next-server/server/image-optimizer.ts

```diff
@@ -211,7 +211,7 @@ export async function imageOptimizer(
 
       upstreamBuffer = Buffer.concat(resBuffers)
       upstreamType = mockRes.getHeader('Content-Type')
-      maxAge = getMaxAge(mockRes.getHeader('Cache-Control'))
+      maxAge = getMaxAge(mockRes.getHeader('Cache-Control'), imageData.maxAge)
```

Comment:
Let's destructure imageData above like the other properties.

We'll need to pass the value to both places we call `getMaxAge()`